jarbas/dashboard/urls.py

```diff
@@ -1,6 +1,6 @@
 from django.conf.urls import url
-from jarbas.dashboard.sites import dashboard
+from jarbas.public_admin.sites import public_admin
```

Comment:
🎉 shouldn't this and line 1 be separated by one blank line?

Reply:
Fixed in 94d8f94